src/main/java/com/redhat/devtools/lsp4ij/client/features/EditorBehaviorFeature.java

```diff
@@ -110,6 +110,17 @@ public boolean isEnableSemanticTokensFileViewProvider(@NotNull PsiFile file) {
         return true;
     }
 
+    /**
+     * Whether or not the document symbols-based breadcrumbs info provider is enabled.
+     *
+     * @param file the file
+     * @return true if the document symbols-based breadcrumbs info provider is enabled; otherwise false
+     */
+    public boolean isEnableDocumentSymbolsBreadcrumbsInfoProvider(@NotNull PsiFile file) {
```

Comment:
Feature flag defaulting to _enabled_.

Reply:
What about having a new LSPBreadcrumbFeatures?